views/js/runner/plugins/security/preventCopy.js

```diff
@@ -117,6 +117,26 @@ define([
         }
     }
 
+    function toKeyCode(combination) {
+        var keys = combination.split('+');
+        var keyCode = 0;
+
+        _.forEach(keys, function (key) {
+            switch (key) {
+                case 'Ctrl':
+                case 'Meta':
+                    keyCode += 1114112; // @see CKEDITOR.CTRL
```

Comment:
can we use `CKEDITOR.CTRL` in code? instead of `1114112`?

Reply:
Not until the editor is loaded and a global `CKEDITOR` variable assigned.